### ISSUE TITLE
Inheritance graph at https://dontcallmedom.github.io/webidlpedia/inheritance.html broken

### ISSUE THREAD

**Issue by cpcallen** [dontcallmedom/webidlpedia#1]
The page appears blank except for the title; in the JS console appears:

```
Uncaught TypeError: json.filter is not a function
    at inheritance.js:25
    at d3.v2.js?2.9.3:2950
    at ready (d3.v2.js?2.9.3:2940)
    at XMLHttpRequest.req.onreadystatechange (d3.v2.js?2.9.3:2933)
```

(This is in Chrome 60.0.3112.113.)

**Comment by altayhunter** [comment]
It's now broken because results.json is missing.
```
GET https://dontcallmedom.github.io/webidlpedia/results.json 404
```
```
Uncaught TypeError: Cannot read property 'results' of null
    at inheritance.js:25
    at d3.v2.js?2.9.3:2950
    at ready (d3.v2.js?2.9.3:2940)
    at XMLHttpRequest.req.onreadystatechange (d3.v2.js?2.9.3:2933)
```

**Comment by dontcallmedom** [comment]
closed by #4
